tasks/configure.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Deploy config from template
+  become: yes
+  template:
+    dest: "{{ prometheus_mqtt_exporter.path.install }}/prometheus_mqtt_exporter/config.yml"
+    group: "{{ prometheus_mqtt_exporter.group }}"
+    mode: 0644
+    owner: "{{ prometheus_mqtt_exporter.user }}"
+    src: config.yml.j2
+  notify:
+    - restart exporter
```

Comment:
This handler name is too generic, there's a high chance it could trigger another exported when included in another role with multiple exporters, perhaps ```restart mqtt-export``` might be better?

Reply:
changed.